experimental/dds/tree2/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -223,6 +224,7 @@ export function areOutputCellsEmpty(mark: Mark<unknown>): boolean {
 	switch (type) {
 		case NoopMarkType:
 		case "Insert":
+		case "Placeholder":
```

Comment:
Can't a Placeholder mark have a populated `detachEvent` field, indicating that the cells are empty?
I imagine that would hapeen when composing a move and its inverse after rebasing them both over the deletion of the moved content.